packages/react/src/components/Popover/index.tsx

```diff
@@ -253,8 +254,8 @@ const Popover = forwardRef<HTMLDivElement, PopoverProps>(
             role="dialog"
             style={styles.popper}
             {...attributes.popper}
-            {...props}
             {...additionalProps}
+            {...props}
```

Comment:
This appears to be a bug - author provided props should always override internal props.